### PR TITLE
Define DSL1 explicitly, as Nextflow made DSL2 default

### PR DESCRIPTION
**Pull request by Lehmann-Fabian**
Signed-off-by: Lehmann_Fabian <fabian.lehmann@informatik.hu-berlin.de>

Cannot start DSL1 workflows with the newest Nextflow version. So I had to define the DSL version explicitly.

## PR checklist

- [ x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](<https://github.com/>nf-core/eager/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/eager _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by jfy133 — APPROVED** [review]
Thanks @Lehmann-Fabian ! I'm just going to do a quick local test what happens when you run this but with earlier versions of NXF and then merge

**Comment by jfy133** [comment]
Ah, please update changelog then we are good to go! :)

**Comment by Lehmann-Fabian** [comment]
Thanks a lot for your review. I updated the changelog accordingly.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,13 +3,26 @@
...
+### `Deprecated`
+
 ## [2.4.4] - 2022-04-08
 
 ### `Added`
 
 ### `Fixed`
 
 - Fixed some auxiliary files (adapater list, snpcapture/pileupcaller/sexdeterrmine BED files, and pileupCaller SNP file, PMD reference mask) in some cases only be used against one sample (❤ to @meganemichel for reporting, fix by @jfy133)
+- [#882](https://github.com/nf-core/eager/pull/882) Define DSL1 explicitly, as Nextflow made DSL2 default
```
Comment (jfy133) [inline CHANGELOG.md:25]:
```suggestion
```

**Pull request merged**